carmin-server/server/resources/models/execution.py

```diff
@@ -33,9 +33,9 @@ class Execution():
     """
 
     def __init__(self,
-                 name: str,
-                 pipeline_identifier: str,
-                 input_values: object,
+                 name: str = None,
```

Comment:
At first I didn't want to allow none for these values as they are illegal and should never be None, but as we always do the transfer via the Schema and it performs the validation this is ok. I will change my call to this constructor in the POST.

Reply:
This was my thought as well. I don't think it causes any issues, and remains safe.